### PR TITLE
Feature/137 update epa template

### PR DESCRIPTION
**Pull request by maxdiebold-erg**
## Related Issues:
* [HMW-137](https://jira.epa.gov/browse/HMW-137)

## Main Changes:
* Updated the base HMTL template to the current one stipulated by EPA.
* Included all necessary EPA files locally.
* Addressed styles that were broken by the update.

## Steps To Test:
1. Test as many pages as possible and verify the appearance of all non-template components are the same as before.
2. Ensure the template is updated on the `400.html` and `500.html` pages.

## NOTES:
All of the auxiliary template files are located under `public/epa-template-files` in both the server and client, so that they could be easily copied/synced. The duplicates will be consolidated when building the application, but there could be a better way to make the files reachable by React during development (I considered symlinks, but I don't think they would play well between operating systems).

### CODE REVIEW

**Review by cschwinderg — CHANGES_REQUESTED** [review]
The header and footer looks good.

Here are some differences that I found (left = new and right = production).

* Fonts in tables in popups seem a to large.
image

* There is some extra padding on the search in menu and search suggestions menu.
image

File: app/client/public/site.webmanifest [hunk 1]
```diff
@@ -3,12 +3,12 @@
   "name": "How’s My Waterway?",
   "icons": [
     {
-      "src": "/android-chrome-192x192.png",
+      "src": "/images/android-chrome-192x192.png",
```
[the remark this answers is not in the document]
Comment (maxdiebold-erg) [reply app/client/public/site.webmanifest:6]: Good catch, missed updating that file when adding that subdirectory.

**Comment by maxdiebold-erg** [comment]
I just realized the new template changed the `html` tag's font size, so any `rem`-sized styles will be a bit off. It was previously set at 106.25%, i.e. `17px`, now it is at the default `16px`. I'm not sure if we want to account for this or roll with the new size. The disclaimer modal font seems a bit small.

**Review by cschwinderg — APPROVED** [review]
Looks good! Thanks for making those changes.

**Pull request merged**